### DISCUSSION TITLE
Single-Page Load Functionality

### DISCUSSION THREAD

**Discussion by mk5-danny-w** [Laravel-Backpack/community-forum#1228] [Q&A (Help)] [answered]
Hi Laravel Backpack Team,

I'm currently evaluating Laravel Backpack packages and noticed in your demo site (https://demo.backpackforlaravel.com/) that the page transitions seem to function as single-page loads. I am interested in understanding if this feature is available across all packages or if it is exclusive to a specific subscription.

Could you please clarify if the single-page load functionality is included as a core feature of your packages, or if there are particular implementation required?

Thanks

[a comment by an automated account is not shown]

**Comment by mk5-danny-w** [comment]
Hi. 
I would like to receive the specific answer to this concern as this will be the decision maker to buy the backpack package.
Please pass this to the support team.
Thanks

**Comment by jcastroa87** [comment, marked as answer]
Hi @mk5-danny-w,

Thanks for reaching out and for your interest in Backpack!

Backpack is not built as a single-page application (SPA); instead, it operates with server-rendered pages, meaning each page load fetches a new view from the server. However, certain features within Backpack use AJAX (like in-line editing, filters, and some forms) to enhance interactivity and reduce full-page reloads where possible.

The fast page loads you’ve noticed are actually thanks to Basset, Backpack’s own caching system. Basset optimizes asset loading and minimizes load times across pages, creating a smoother and faster user experience that can feel somewhat like an SPA. This feature is part of Backpack’s core.

If you're looking for a fully SPA-like experience, it might require some customizations beyond Backpack's built-in functionality. Let us know if you have any more questions about specific use cases or customization options!

Cheers.
